src/hardware/siglent-sds/api.c

```diff
@@ -155,7 +155,7 @@ static const uint64_t averages[] = {
 
 /* Do not change the order of entries. */
 static const char *data_sources[] = {
-	"Display",
+	"Screen",
```

Comment:
config_get for DATA_SOURCE returns "Screen" to the cli/pulseview, and since the enum is also DATA_SOURCE_SCREEN I decided to change these two occurrences of "Display" into "Screen".